changelog.md

```diff
@@ -1,3 +1,32 @@
+# 4.2.0 / YYYY-MM-DD
+
+### New Features
+
+- Handle right-to-left (rtl) languages on interface
+- Introduce arab language selection, and add arabic translation via Weblate
+- Notification can be marked as read without redirection
+- UI/UX improvement (responsive, buttons, redirections)
```

Comment:
Do not hesitate to order by more important in the beggining. I think (I'm not sure) UI/UX improvement is more important/visible that the notification feature to mark as read.